README.md

```diff
@@ -19,6 +19,7 @@ To contribute:
 
 | Name                                                                                                                                          | Location                                | Application Period    | Notes                                                                                          | Date Posted |
 | --------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------- | --------------------- | ---------------------------------------------------------------------------------------------- | ----------- |
+| [Ribbit](https://flyribbit.applytojobs.ca/engineering/26524)                                                                                  | Toronto                                 | Unknown               | Software Engineering Intern - Robotics                                                         | Nov 23      |
```

Comment:
Maybe put `Rolling Applications` in the Application Period field instead of `Unknown`?

Reply:
Good idea! I can do this!